ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/individual-journey.spec.ts

```diff
@@ -194,4 +201,11 @@ describe('IndividualJourney', () => {
   it('should throw an exception if proceeding without having entered the journey', () => {
     expect(() => journey.next()).toThrowError('Journey must be entered before navigation is allowed');
   });
+
+  it('it should check for drop off points when next is clicked', () => {
```

Comment:
rather than check the code here, what do you think about testing the behaviour such as checking the effects of the drop point happening?

for example, if proceeding with the internet question answered negatively, we expect to be redirected to ThankYou and `isSubmitted` is true?

Reply:
removed this test as its covered in the drop off points tests.